recipe/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set name = "uamqp" %}
+{% set version = "1.6.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/azure-{{ name }}-python/archive/v{{ version }}.tar.gz
+  sha256: 1f42dc85940d11f3af553551bdd0940a85fff8fa49ceece6e1e969e0279944a1
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # py<37 - https://github.com/Azure/azure-uamqp-python/blob/main/setup.py#L294
+  skip: True  # [py<37]
+  # cython <0.30 not found
+  skip: True  # [s390x or py>312]
+  # openssl<3.* is not avaliable
+  skip: True  # [linux and py>311]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # The CXX compiler identification is unknown
+    - {{ compiler('cxx') }}  # [linux]
+    - cmake
+    # CMake Error: CMake was unable to find a build program corresponding to "Unix Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
+    - make  # [linux]
+  host:
+    - python
+    # https://github.com/Azure/azure-uamqp-python/blob/v1.6.11/build_many_linux.sh#L19
+    - cython <0.30
+    - setuptools >=42
+    - wheel
+    - pip
+    - openssl 3.0.15  # [linux]
```

```suggestion
    - openssl {{ openssl }}  # [linux]
```
It's better for new Python build-outs, for example, for the future py314

---

done